src/app/chat/views/message_list.nim

```diff
@@ -37,24 +39,30 @@ QtObject:
     if index.row < 0 or index.row >= self.messages.len:
       return
     let message = self.messages[index.row]
+    let repeatMessageInfo = (index.row == 0) or message.fromAuthor != self.messages[index.row - 1].fromAuthor
```

Comment:
Would've preferred doing this inside QML but I didn't manage to make `ChatMessageList.messages` to be readable using `QtProperty`. Decided then to calculate this in the `data` function and just expose a boolean to QML.